### DISCUSSION TITLE
bfe没有转发规则的站点状态码定制需求

### DISCUSSION THREAD

**Discussion by lf2186** [bfenetworks/bfe#958] [Ideas（新功能的建议和讨论）]
server {
      listen 80 default_server;

      root /var/www/default;
      index index.html index.htm;
      server_name _;
      return 404;
      access_log off;

    }

hi, 您好，

    上面我们是针对nginx默认站点配置的返回码配置，结果是凡是没有配过servername的域名 和 ip访问，都返回404，而不是500.

便于日志统计整个服务器服务可用性。（可用性，我们是用（1-5xx错误量/总量）*100%计算）

请问，bfe是否可以支持类似的功能。

**Comment by ohscartao** [comment]
你好，BFE目前的框架确实是不支持定制返回码。找不到租户这种情况，定制返回码，这个需求确实会有用。我会提交一个issue

**Reply by mileszhang2016** [reply]
https://github.com/bfenetworks/bfe/issues/972

**Comment by iyangsj** [comment]
@lf2186 参见 [mod_errors ](https://www.bfe-networks.net/zh_cn/modules/mod_errors/mod_errors/) 可替换这类异常响应为自定义响应
